src/test/java/uk/ac/ebi/embl/flatfile/writer/embl/EmblReducedFlatFileWriterTest.java

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright 2012 EMBL-EBI, Hinxton outstation
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package uk.ac.ebi.embl.flatfile.writer.embl;
+
+import uk.ac.ebi.embl.api.entry.EntryFactory;
+import uk.ac.ebi.embl.api.entry.Text;
+import uk.ac.ebi.embl.api.entry.feature.Feature;
+import uk.ac.ebi.embl.api.entry.feature.FeatureFactory;
+import uk.ac.ebi.embl.api.entry.location.LocationFactory;
+import uk.ac.ebi.embl.api.entry.qualifier.QualifierFactory;
+import uk.ac.ebi.embl.api.entry.reference.Article;
+import uk.ac.ebi.embl.api.entry.reference.Reference;
+import uk.ac.ebi.embl.api.entry.reference.ReferenceFactory;
+import uk.ac.ebi.embl.api.entry.sequence.Sequence.Topology;
+import uk.ac.ebi.embl.api.entry.sequence.SequenceFactory;
+import uk.ac.ebi.embl.api.validation.Severity;
+import uk.ac.ebi.embl.api.validation.ValidationResult;
+import uk.ac.ebi.embl.flatfile.FlatFileUtils;
+import uk.ac.ebi.embl.flatfile.reader.embl.EmblLineReader;
+import uk.ac.ebi.embl.flatfile.reader.embl.RAReader;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+
+public class EmblReducedFlatFileWriterTest extends EmblWriterTest {
+
+	public void testWrite_Entry() throws IOException {
```

Comment:
- Can we have one test for CON with contigs + features + sequence -> check flat file
- Can we have one test for non-CON with features + sequence -> check flat file

- Can we have one test for CON with NO contigs + features + sequence -> check that this results in error
- Can we have one test for non-CON with contigs + features + sequence -> check that this results in error

- Can we have one test without sequence -> check that this results in error
- Can we have one test without features ->  check flat file